scout/server/blueprints/variant/utils.py

```diff
@@ -453,6 +453,8 @@ def frequency(variant_obj):
         variant_obj.get("exac_frequency") or 0,
         variant_obj.get("gnomad_frequency") or 0,
         variant_obj.get("gnomad_mt_homoplasmic_frequency") or 0,
+        variant_obj.get("swegen_alu") or 0,
```

Comment:
```suggestion
```

Reply:
If you want them, it should be all of `alu`, `sva`, `l1` and `herv`, but as long as the mei_max works, there should be no reason to have any of the individual ones!